internal/service/sagemaker/domain.go

```diff
@@ -1223,10 +1269,50 @@ func expandDomainSettingsUpdate(l []interface{}) *sagemaker.DomainSettingsForUpd
 
 	config := &sagemaker.DomainSettingsForUpdate{}
 
+	if v, ok := m["docker_settings"].([]interface{}); ok && len(v) > 0 {
+		config.DockerSettings = expandDockerSettings(v)
+	}
+
 	if v, ok := m["execution_role_identity_config"].(string); ok && v != "" {
 		config.ExecutionRoleIdentityConfig = aws.String(v)
 	}
 
+	if v, ok := m["security_group_ids"].(*schema.Set); ok && v.Len() > 0 {
```

Comment:
add tests for updating these values please

Reply:
Thank you for speedy review @DrFaust92. Have streamlined existing test case and renamed from testAccDomain_securityGroup to testAccDomain_defaultUserSettingsSecurityGroupUpdated.
( Removed testAccDomainConfig_securityGroup1 function )


$ make testacc TESTS=TestAccSageMaker_serial/Domain/defaultUserSettingsSecurityGroupUpdated PKG=sagemaker
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/sagemaker/... -v -count 1 -parallel 20 -run='TestAccSageMaker_serial/Domain/defaultUserSettingsSecurityGroupUpdated'  -timeout 360m
=== RUN   TestAccSageMaker_serial
=== PAUSE TestAccSageMaker_serial
=== CONT  TestAccSageMaker_serial
=== RUN   TestAccSageMaker_serial/Domain
=== RUN   TestAccSageMaker_serial/Domain/defaultUserSettingsSecurityGroupUpdated
--- PASS: TestAccSageMaker_serial (398.48s)
    --- PASS: TestAccSageMaker_serial/Domain (398.48s)
        --- PASS: TestAccSageMaker_serial/Domain/defaultUserSettingsSecurityGroupUpdated (398.48s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/sagemaker  424.234s